rcgen/prettier/patchers.js

```diff
@@ -0,0 +1,20 @@
+// @ts-check
+
+const {ignoreFilesWithGit} = require('../git/utils');
+const {
+  configureVscode,
+  excludeFilesFromSearchInVscode,
+  excludeFilesInVscode,
+  recommendVscodeExtensions
+} = require('../vscode/utils');
+const {prettierFiles} = require('./files');
+
+const prettierFilenames = prettierFiles.map(({filename}) => filename);
+
+exports.prettierPatchers = [
+  ignoreFilesWithGit(...prettierFilenames),
+  excludeFilesFromSearchInVscode(...prettierFilenames),
+  excludeFilesInVscode(...prettierFilenames),
+  recommendVscodeExtensions('esbenp.prettier-vscode'),
+  configureVscode({'editor.formatOnSave': true})
```

Comment:
why is this something that prettier provides? Isn't it distinct from prettier whether or not I would want my editor to format on save?